src/main/java/me/champeau/gradle/japicmp/JApiCmpWorkerAction.java

```diff
@@ -373,7 +374,7 @@ private void generateOutput(JarArchiveComparator jarArchiveComparator) {
             } catch (URISyntaxException e) {
                 reportLink = null;
             }
-            StringBuilder message = new StringBuilder("Detected binary changes.\n")
+            StringBuilder message = new StringBuilder("Verification failed: Detected binary changes.\n")
```

Comment:
Do we have to add this string? Looks that the verification is already encoded in the exception type.